app/views/admin/settings/project_life_cycle_definitions/form.html.erb

```diff
@@ -27,12 +27,16 @@ See COPYRIGHT and LICENSE files for more details.
 
 ++#%>
 
-<% heading_scope = @definition.persisted? ? :edit : :new %>
+<% definition_heading = if @definition.persisted?
+                          @definition.name
+                        else
+                          t("settings.project_phase_definitions.new.heading")
+                        end %>
 
 <% html_title t(:label_administration),
               t("settings.project_phase_definitions.heading"),
-              t("settings.project_phase_definitions.#{heading_scope}.heading") %>
+              definition_heading %>
 
-<%= render Settings::ProjectLifeCycleStepDefinitions::FormHeaderComponent.new(@definition, heading_scope:) %>
+<%= render Settings::ProjectLifeCycleStepDefinitions::FormHeaderComponent.new(@definition, definition_heading:) %>
```

Comment:
Why not place:

```
definition_heading = if @definition.persisted?
                          @definition.name
                        else
                          t("settings.project_phase_definitions.new.heading")
                        end
```

inside the `Settings::ProjectLifeCycleStepDefinitions::FormHeaderComponent` componet? It would be one less parameter to pass and you could have it dealt with inside the component code which means less code in a view/template.

Reply:
Good point, however that also involves moving the `<% html_title %> ` helper call, which may be better placed in the root template. However, since the component is also called `FormHeaderComponent` then it's a good place to have the title helper called in.